templates/header/topbar_end.html

```diff
@@ -24,11 +24,6 @@
                     #}<li class="pure-menu-item pure-menu-has-children">
                         <a href="#" class="pure-menu-link" aria-label="Rust">Rust</a>
                         <ul class="pure-menu-children">
-                            {{ macros::menu_link(
-                                href="https://www.rust-lang.org/",
-                                text="Rust website",
```

Comment:
I think it makes sense to make an exception here, since it's the official site for the language.

Reply:
:+1: